platform/linux-generic/odp_packet_io.c

```diff
@@ -150,6 +150,8 @@ static void init_pktio_entry(pktio_entry_t *entry)
 	init_out_queues(entry);
 
 	pktio_classifier_init(entry);
+
+	odp_pktio_config_init(&entry->s.config);
```

Comment:
Indeed. It was fixed in the meantime by you in „linux-gen: packet: remove lazy parsing” ce8bebad45c590ad5b06d31d8be9fae8182f944a